File: javascript/01_DynamicPrograming/02_Fibonacci_tabulation.js
function fib(n){
    let first = 0;
    let second = 1;
    let res = 0;
    for(let i=2;i<n+1;i++){
        res = first + second;
        first = second;
        second = res; 
    }
    return res;
}
let res = fib(5)
console.log(res);

File: python/DynamicPrograming/02_Fibonacci_tabulation.py
n = 5
def fib(n):
    first = 0
    second = 1
    res = 0
    for i in range(2,n+1):
        res = first + second 
        first = second 
        second = res 
    return res 

res = fib(n)
print(res)